setup.cfg

```diff
@@ -44,6 +44,12 @@ addopts = --doctest-rst
 console_scripts =
     run_dgfit = dgfit.run_dgfit:main
     plot_dgfit = dgfit.plotting.plot_dgfit:main
+    #plot_dustmodel = dgfit.plotting.plot_dustmodel:main
```

Comment:
Is there a reason why plot_dustmodel is no longer possible?

Reply:
Yes, plot_dustmodel makes plots that are already included in plot_dgfit. I think this was a preliminary version of plot_dgfit. I didn't want to delete the whole file since I don't know if it should be stored somewhere or not. The previous comments about this file are correct but I didn't look into them since I just moved the script to the 'old' directory.

Reply:
Please then remove the commented out line.  We can always add it back when needed.